packages/common/src/search/utils.ts

```diff
@@ -292,6 +292,7 @@ export function migrateToCollectionKey(
  * @param scope Catalog scope to search through
  * @returns The first predicate with a `group` field (if present)
  */
+// istanbul ignore next
```

Comment:
if you end up pushing any more commits I suggest
```suggestion
// istanbul ignore next - deprecated
```